MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include ts/frontend/model-server.jar
 include PyPiDescription.rst
 include ts/configs/*
+include ts/version.txt
```

Comment:
Can't we keep version.txt at root level?

Reply:
we are having version.txt for model archiver as well and keep the files under ts folder while packaging its placed in ts folder.
and version.txt related to model archiver is placed in model archiver folder